upb/upb.c

```diff
@@ -67,6 +67,18 @@ static void *upb_global_allocfunc(upb_alloc *alloc, void *ptr, size_t oldsize,
   }
 }
 
+static uint32_t* upb_cleanup_pointer(void* cleanup_metadata) {
```

Comment:
upb puts pointers on the right, like LLVM. I should add a `.clang-format` for this.

Reply:
Done